hack/verify-examples.sh

```diff
@@ -28,41 +28,49 @@ else
     kubectl apply -f deploy/example/storageclass-azuredisk-csi.yaml
 fi
 
-if [[ "$1" == "linux" ]]; then
-    kubectl apply -f deploy/example/deployment.yaml
-    kubectl apply -f deploy/example/statefulset.yaml
-    kubectl apply -f deploy/example/statefulset-nonroot.yaml
-    if [[ "$#" -gt 2 ]]&&[[ "$3" == *"ephemeral"* ]]; then
-        kubectl apply -f deploy/example/nginx-pod-ephemeral.yaml
-        kubectl apply -f deploy/example/daemonset-azuredisk-ephemeral.yaml
+rollout_and_wait() {
+    echo "Applying config \"$1\""
+    trap "echo \"Failed to apply config \\\"$1\\\"\" >&2" err
+
+    APPNAME=$(kubectl apply -f $1 | grep -E "^(:?daemonset|deployment|statefulset|pod)" | awk '{printf $1}')
```

Comment:
if it could be parallel, that would save time, not sure how to do that in shell. let's merge this PR first, thanks.